install/Knative-with-Minishift.md

```diff
@@ -20,7 +20,14 @@ You can find [guides for other platforms here](README.md).
 minishift version
 ```
 
-## Configure and start minishift
+## Run automated set-up script
```

Comment:
One comment, how about update the TOC as follows:

## Automatic Set Up
xxx
## Manually Set Up

### Manually configure and start minishift

### Configuring oc (openshift cli)

Reply:
done what you suggested (I think; please check & Resolve Conversation if OK for you now).